app/controllers/account/billing/invoices/list.js

```diff
@@ -133,6 +133,105 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
           cellComponent   : 'ui-table/cell/events/cell-action'
         }
 
+      ];
+    } else if (this.model.params.invoice_status === 'unpaid') {
+      columns = [
+        {
+          name      : 'Invoice ID',
+          valuePath : 'identifier'
+        },
+        {
+          name          : 'Event Name',
+          valuePath     : 'event',
+          cellComponent : 'ui-table/cell/events/cell-event-invoice'
+        },
+        {
+          name            : 'Amount',
+          valuePath       : 'amount',
+          extraValuePaths : ['event'],
+          cellComponent   : 'ui-table/cell/events/cell-amount',
+          isSortable      : true,
+          headerComponent : 'tables/headers/sort'
+        },
+        {
+          name            : 'Status',
+          valuePath       : 'status',
+          isSortable      : true,
+          headerComponent : 'tables/headers/sort'
+        },
+        {
+          name            : 'Action',
+          valuePath       : 'identifier',
+          extraValuePaths : ['status'],
+          cellComponent   : 'ui-table/cell/events/cell-action'
+        }
+
+      ];
```

Comment:
I don't understand. It's the exact same code. Why are there else if statements everywhere?

Reply:
> I don't understand. It's the exact same code. Why are there else if statements everywhere?

it will be changed, I don't know what would be the exact column name. Not mentioned in issue.